var/spack/repos/builtin/packages/py-javaproperties/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyJavaproperties(PythonPackage):
+    """Read & write Java .properties files."""
+
+    homepage = "https://github.com/jwodder/javaproperties"
+    url      = "https://pypi.io/packages/source/j/javaproperties/javaproperties-0.7.0.tar.gz"
+
+    version('0.7.0', sha256='cf45b39fcbaeced1dfc0b7f2bda16e34fc0349116269e001dada42fd2e145d87')
+    version('0.5.1', sha256='2b0237b054af4d24c74f54734b7d997ca040209a1820e96fb4a82625f7bd40cf')
+
+    depends_on('python@2.7:2.8,3.4:3.999', type=('build', 'run'))
```

Comment:
Should this also be a link dependency?

Reply:
Nope, like most Python packages this is a pure-Python library with no `*.so` files in the installation directory. The build log shows no evidence of linking to any libraries or header files.